docs/examples/gluster-s3-storage-template/README.md

```diff
@@ -2,111 +2,132 @@
 
 ## Prerequisites
 
-  OpenShift setup is up with master and nodes ready.
+OpenShift setup is up with master and nodes ready.
 
-  cns-deploy tool is ran and heketi service is ready
+cns-deploy tool is ran and heketi service is ready.
```

Comment:
Change the above two lines to a bulleted list, and update the second line to `cns-deploy tool has been run`

Reply:
Done